Open_Foam/Post_Process.ipynb

```diff
@@ -4,129 +4,112 @@
+    "The general decreasing trend of the residual and error values confirm that the numerical solution of the conservation/governing equations are converging to specific final values. This decreasing trend is the first necessary, but not sufficient, general criteria to confirm the convergence of the CFD simulation.\n",
+    "\n",
+    "The second step of post-processing is to visualize dimensionless form of the important flow field variables within the CFD domain. This visualization step can be problem specific and should be performed over an adequate plane of choice, which will be explained for the current case study shortly. To post-process the obtained numerical results in OpenFoam it is recommended to use the post-processor package of OpenFoam called `ParaView`. `ParaView` can be launched running the command\n",
+    "\n",
+    "```C++\n",
+    "paraFoam\n",
     "```\n",
     "\n",
-    "<img src=\"./Images/sample.jpg\" width=\"500\">\n",
+    "in the terminal, while locating in the working directory of the simulation. Running this command will simply read all the obtained numerical results and get it ready for post-processing. Once the user runs `paraFoam` command and the software loads up, all the boundaries and fields should be activated (i.e. checked in their box) and the `Apply` bottom under `Properties` window should be hit to load the numerical results.\n",
+    "\n",
+    "For most of the problems visualization of normalized velocity or pressure contours over a horizontal or vertical plane at the center of the CFD domain is a good start point for post-processing the obtained numerical data. For the current problem, since the entire domain is two dimensional and planar, the entire domain itself is the best candidate of post-processing plane. User should now define the normalized variable of interest needs to be visualized. Defining variables can be done following path of `Filters/Data Analysis/Calculator`. This command will open a new panel under `Properties`, where users can define their variable of interest; Name the variable (i.e. `U_norm`) and enter the required formula/definition for the normalized variable of interest (i.e. `U_x/1`) and hit `Apply`. This will calculate the normalized streamwise velocity with free stream velocity within the entire CFD domain for all the time steps and visualize the results for `t=0`. Using the `play` bottom users can see the formation of laminar boundary layer and wake around and in the back of the rotating cylinder:\n",
+    "\n",
+    "<img src=\"./Images/2velCountor.png\" width=\"700\" align=\"middle\">\n",
     "</br>\n",
-    "Fig. 1 - The caption for the image goes here."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Insert a YouTube video Available on the SFO Channel \n",
+    "Fig. 2 - Normalized streamwise velocity contours with the free stream velocity for flow over a rotating cylinder with \n",
+    "$Re=100$ and $TSR$=1 at `t=150`.\n",
```

Comment:
First, does this results represent Re=100 and TSR=1?
Second do you have a full animation of the flow field? If yes, share it with us and we will upload it on our Youtube channel and exert it here.